src/webgpu/api/validation/image_copy/texture_related.spec.ts

```diff
@@ -2,7 +2,11 @@ export const description = '';
 
```

Comment:
IIUC most of these tests could be expanded to 1d as well. Can you put a TODO in the file description for that?

(I know it's already mentioned in `image_copy/README.txt`, but here we can be more specific.
Also, this partially resolves the TODO in that readme, but I don't know if there's still more 3d texture work to do so it can probably be left as is.)

Reply:
Done. 

For the TODO in README.txt, I will remove the `3d` if I feel like that validation tests for 3D texture copies are complete.